mothernet/evaluation/plot_shape_function.py

```diff
@@ -12,7 +12,7 @@ def plot_shape_function(bin_edges: np.ndarray, w: np.ndarray, feature_names=None
         class_range = [0]
         columns = min(int(np.sqrt(num_features)), 6)
         rows = int(np.ceil(num_features / columns))
-    fig, axs = plt.subplots(rows, columns, figsize=(2*rows, 2*columns),
```

Comment:
wasn't this already switched in develop? hm let me check if I pushed all my changes.

Reply:
ok sorry something was missing!